docs/api-reference.md

```diff
@@ -3,25 +3,623 @@ id: api-reference
+#### Example
+
+<!-- prettier-ignore-start -->
+
+[mdis]:# (../tests/saga-sample/saga.js#execute)
+```js
+await sideEffects.executeCommand({
+  aggregateName: 'User',
+  aggregateId: event.aggregateId,
+  type: 'requestConfirmUser',
+  payload: event.payload
+})
+```
+
+<!-- prettier-ignore-end -->
+
+### scheduleCommand
+
+Similar to `executeCommand`, but delays command execution until a specified moment in time.
+
+#### Arguments
+
+| Argument Name | Description                                                                                                        |
+| ------------- | ------------------------------------------------------------------------------------------------------------------ |
+| command       | Specifies a command object. Refer to the [Write Side](write-side#sending-a-command) document for more information. |
+
+#### Example
+
+<!-- prettier-ignore-start -->
+
+[mdis]:# (../tests/saga-sample/saga.js#schedule)
+```js
+await sideEffects.scheduleCommand(
+  event.timestamp + 1000 * 60 * 60 * 24 * 7,
+  {
+    aggregateName: 'User',
+    aggregateId: event.aggregateId,
+    type: 'forgetUser',
+    payload: {}
+  }
+)
+```
+
+<!-- prettier-ignore-end -->
+
+## Client-Side API
+
+### HTTP API
+
+Resolve provides a standard HTTP API that allows you to send aggregate commands and query Read Models and View Models. Refer to the [Standard HTTP API](curl.md) document for more information.
+
+#### Read Model API
+
+You can query a Read Model from the client side by sending a POST request to the following URL:
+
+```
+http://{host}:{port}/api/query/{readModel}/{resolver}
+```
+
+##### URL Parameters:
+
+| Name          | Description                                                                                                                           |
+| ------------- | ------------------------------------------------------------------------------------------------------------------------------------- |
+| **readModel** | The Read Model name as defined in [config.app.js](https://github.com/reimagined/resolve/blob/master/examples/with-saga/config.app.js) |
+| **resolver**  | The name of a [resolver defined in the Read Model](#resolvers)                                                                        |
+
+The request body should have the `application/json` content type and the following structure:
+
+```js
+{
+  param1: value1,
+  param2: value2,
+  // ...
+  paramN: valueN
+}
+```
+
+The object contains the parameters that the resolver accepts.
+
+##### Example
+
+Use the following command to get 3 users from the [with-saga](https://github.com/reimagined/resolve/tree/master/examples/with-saga) example.
 
 ```sh
-$ curl -X POST http://localhost:3000/api/commands/ \
+curl -X POST \
+-H "Content-Type: application/json" \
+-d "{\"page\":0, \"limit\":3}" \
+"http://localhost:3000/api/query/default/users"
+```
+
+#### View Model API
+
+You can query a View Model from the client side by sending a POST request to the following URL:
```

Comment:
Do not use -ing forms

Reply:
'To query a View Model from the client side, send a GET request to the following URL:'

Reply:
Ok